tests/ocf-meta-inf_inc/EPUB/package.opf

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
+  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <dc:coverage>Open Container Format</dc:coverage>
+    <dc:creator>Zheng Xu</dc:creator>
+    <dc:date>2022-05-28</dc:date>
+    <dc:description>An extra config file inside meta-inf folder.</dc:description>
+    <dc:identifier id="pub-id">ocf-meta-inf_inc</dc:identifier>
+    <dc:language>en</dc:language>
+    <dc:title>Test of extra config file from meta-inf folder but not in reserved files list</dc:title>
```

Comment:
Title is too long for my taste... can we make it shorter to something like

> Extra configuration file in meta-inf

Reply:
Sounds good. Updated